openforcefield/typing/engines/smirnoff/parameters.py

```diff
@@ -2519,14 +2937,9 @@ class ChargeIncrementType(ParameterType):
         """
         _VALENCE_TYPE = 'Bond'  # ChemicalEnvironment valence type expected for SMARTS
         _ELEMENT_NAME = 'ChargeIncrement'
-        _REQUIRED_SPEC_ATTRIBS = ['smirks', 'chargeIncrement']
-        _REQUIRE_UNITS = {
-            'chargeIncrement': unit.elementary_charge
-        }
-        _INDEXED_ATTRIBS = ['chargeIncrement']
 
-        def __init__(self, node, parent):
-            super().__init__(**kwargs)
+        charge_increment = IndexedParameterAttribute(unit=unit.elementary_charge)
```

Comment:
Thanks for standardizing this -- The switch to `charge_increment` will make this consistent with fields like `aromaticity_model` and `combining_rules`. We'll need to update the SMIRNOFF spec docs with the new syntax (Oh boy, this means a `ChargeIncrement` section version update too!)

Reply:
Ah! This was kind of an accident. I can revert it for this PR and we can change it later if you prefer.

Reply:
This PR is already big enough without getting a spec change mixed in. Let’s revert this to ‘chargeincrement’ and open an issue to do this later.

Reply:
Sounds good! Done in 6028828.